src/main/java/net/syscon/elite/repository/jpa/model/CourtEvent.java

```diff
@@ -90,4 +90,8 @@
 
     @Column(name = "DIRECTION_CODE", length = 12)
     private String directionCode;
+
+    public LocalDateTime getEventDateTime() {
+        return eventDate.atTime(startTime.toLocalTime());
```

Comment:
Simple helper method to avoid the need to call two separate ones.  Note there are 2 separate fields due to the NOMIS db structure.